libcodechecker/server/server.py

```diff
@@ -728,9 +728,8 @@ def __init__(self,
                                               certfile=ssl_cert_file)
 
             else:
-                LOG.info("SSL key or cert files not found. Cert: " +
-                         ssl_cert_file +
-                         "\n Key: " + ssl_key_file)
+                LOG.info("SSL key or cert files not found (cert: {0}, "
```

Comment:
Perhaps this log message could be improved. It feels weird saying not found and then printing a path.

`Searching for SSL key at {0}, cert at {1}, not found...`?